### ISSUE TITLE
Rejecting command message: MQTT protocol does not support this operation

### ISSUE THREAD

**Issue by FelixXWS** [Azure/azure-iot-sdk-csharp#164]
Hello,

if I try to reject a message ("DeviceClient.RejectAsync(string lockToken)") because of certain failures I get the message: "MQTT protocol does not support this operation". Is this a bug or does the current Mircorosft.Azure Devices.Client-framework (v. 1.5.0) really not support rejecting of messages in combination with MQTT. With AMQP everything works fine. I'm just wondering, because the Azure IoT Suite (Remote Monitoring) uses the same functionality in combination with MQTT. 
If it is not supported, will this be in the near future?

Kind regards,
Felix

**Comment by tameraw** [comment]
@FelixXWS - Per the guidance in this [doc](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support).

"MQTT does not support the reject operations when receiving cloud-to-device messages. If your back-end app needs to receive a response from the device app, consider using direct methods."

**Comment by FelixXWS** [comment]
Hi @tameraw,
thanks for your answer. Found this statement after I posted my question. But I'm wondering why the Iot Suite (Remote monitoring) implements Device Commands with MQTT and reject although it is not supported by the protocol. Are there any plans to support reject, abandon etc. of messages in combination with MQTT? It is very important for us, because we don't have the possiblity to use AQMP in our case.
